### ISSUE TITLE
Registration validation email missing baseUrl.

### ISSUE THREAD

**Issue by edykstra** [BookStackApp/BookStack#218]
**PHP Version:**

```
> php --version;
PHP 5.6.27-0+deb8u1 (cli) (built: Oct 15 2016 15:53:28) 
Copyright (c) 1997-2016 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.0.6-dev, Copyright (c) 1999-2016, by Zend Technologies
```
**MySQL Version:**

I was unable to install MySQL >= 5.6 on Debian Jessie, so I installed MariaDB:

```
> mysql --version;
mysql  Ver 15.1 Distrib 10.0.27-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2
```

**Expected Behavior:**

When registering a new account, and the email address validation email is sent, it should contain the baseUrl() value in the links to that they are clickable.

**Actual Behavior:**

EMail was received, but it contained 'dead' links. Changing the email view to 'Plain Text' displayed this:

```
Email Confirmation

Thanks for joining Some Domain </>.
Please confirm your email address by clicking the button below.

Confirm Email </register/confirm/SubxIprQpCHyZJBNgkdn8ix2>
```

Note there is no "http://www.SomeDomain.com" prefix. As such, the link is not clickable.

**Comment by ssddanbrown** [comment]
@edykstra Thanks for reporting, Good spot. I'll get a bugfix release out over the weekend to cover this and a few other minor bugs.

**Comment by ssddanbrown** [comment]
Thank you for finding this issue. It is now fixed as of release [v0.12.2](https://github.com/BookStackApp/BookStack/releases/tag/v0.12.2).
